tests/infrastructure/utils.py

```
@@ -0,0 +1,57 @@
+    Args:
+        client (DynamicClient): Kubernetes dynamic client used to query cluster resources.
+
+    Raises:
+        ResourceMissingFieldError: If required fields are missing.
+        ResourceValueError: If cpuManagerPolicy has wrong value.
+    """
+    LOGGER.info("Verifying cluster has nodes with NUMA topology and static CPU manager policy")
+    for config in KubeletConfig.get(client=client):
+        kubelet_config = getattr(config.instance.spec, "kubeletConfig", None)
+        if not kubelet_config:
+            raise ResourceMissingFieldError(f"KubeletConfig '{config.name}' missing spec.kubeletConfig")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)